core/src/main/scala/cats/data/OneAnd.scala

```diff
@@ -99,8 +99,22 @@ final case class OneAnd[F[_], A](head: A, tail: F[A]) {
 }
 
 
-private[data] sealed abstract class OneAndInstances extends OneAndLowPriority3 {
+private[data] sealed abstract class OneAndInstances extends OneAndLowPriority4 {
```

Comment:
since we are here, can we fix the priority number to conventional ascending order?